src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,130 @@
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { SearchParams } from '../utils/searchHelper';
 
 export const PeoplePage = () => {
+  const [isLoader, setIsLoader] = useState(true);
+  const [isError, setIsError] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+  const [searchParams] = useSearchParams();
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+  const sex = searchParams.get('sex') || '';
+
+  const toggleByParams = (param: string): SearchParams => {
+    if (param === sort) {
+      if (order === 'desc') {
+        return { sort: null, order: null };
+      }
+
+      return { sort: param, order: 'desc' };
+    }
+
+    return { sort: param };
+  };
+
+  const getClassParams = (param: string) => {
+    if (param === sort) {
+      if (order === 'desc') {
+        return 'fas fa-sort-up';
+      }
+
+      return 'fas fa-sort-down';
+    }
+
+    return 'fas fa-sort';
+  };
+
+  const getCentury = (year: number) => Math.floor(year / 100) + 1;
+
+  const preparePeopleList = [...people]
+    .sort((p1, p2) => {
+      let res = 0;
+
+      switch (sort) {
+        case 'name':
+          res = p1.name.localeCompare(p2.name);
+          break;
+        case 'sex':
+          res = p1.sex.localeCompare(p2.sex);
+          break;
```

Comment:
```suggestion
        case 'name':
        case 'sex':
          res = p1[sort].localeCompare(p2[sort]);
          break;
```
you can group some cases